iphone/cli/commands/_build.js

```diff
@@ -2205,8 +2213,20 @@ iOSBuilder.prototype.validate = function validate(logger, config, cli) {
 								module.isFramework = true;
 								// For Swift frameworks, use the binary inside the .framework for hashing
 								nativeHashes.push(module.hash = this.hash(fs.readFileSync(path.join(module.libFile, this.scrubbedModuleId(module.id)))));
+							} else if (fs.existsSync(path.join(module.modulePath, xcFrameworkOfLib))) {
+								module.libName = xcFrameworkOfLib;
+								module.libFile = path.join(module.modulePath, module.libName);
+								module.isXCFrameworkOfLib = true;
+								//TO DO : Change hash calculation
```

Comment:
<!--
  1 failure:  iphone/cli/comman...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: iphone/cli/commands/_build.js;
  Line: 2220;
-->
- :no_entry_sign: iphone/cli/commands/_build.js line 2220 – Expected space or tab after '//' in comment. (spaced-comment)